pkg/resources/management.cattle.io/v3/globalrolebinding/GlobalRoleBinding.md

```diff
@@ -0,0 +1,11 @@
+## Validation Checks
+
+Note: all checks are bypassed if the GlobalRoleBinding is being deleted
+
+### Escalation Prevention
+
+Users can only create/update GlobalRoleBindings with rights less than or equal to those they currently possess. This is to prevents privilege escalation. 
```

Comment:
```suggestion
Users can only create/update GlobalRoleBindings with rights less than or equal to those they currently possess. This is to prevent privilege escalation. 
```

Reply:
Change Made.